vtStor/DriveManager.cpp

```diff
@@ -53,19 +53,16 @@ eErrorCode cDriveManager::EnumerateDrives( eScanForHardwareChanges ScanForHardwa
     eErrorCode error = eErrorCode::None;
     bool successFlag = false;
 
-    if ( eScanForHardwareChanges::Yes == ScanForHardwareChanges )
-    {
-        // scan for hardware changes
-        m_Drives.clear();
-        m_DevicePaths.clear();
-    }
-    vtStor::GetStorageDevicePaths(m_DevicePaths, eOnErrorBehavior::Continue);
+    m_Drives.clear();
```

Comment:
Updated Issue #4: Drive manager should always clear its drive list when enumerating drives.